packages/flutter/lib/src/material/radio_list_tile.dart

```diff
@@ -316,36 +318,42 @@ class RadioListTile<T> extends StatelessWidget {
         trailing = control;
         break;
     }
+    final ThemeData theme = Theme.of(context);
+    final RadioThemeData radioThemeData = RadioTheme.of(context);
+    final Set<MaterialState> states = <MaterialState>{
+      if (selected) MaterialState.selected,
+    };
+    final Color effectiveActiveColor = activeColor
+      ?? radioThemeData.overlayColor?.resolve(states)
+      ?? theme.toggleableActiveColor;
```

Comment:
So we're not _replacing_ `toggleableActiveColor` so much as enabling apps to replace it (with RadioTheme). This is a reasonable first step however the next step will be more difficult:

- The default value of `effectiveActiveColor` should be based on a ColorScheme color (not `toggleableActiveColor`).  Some apps currently use `Theme.of(context).toggleableActiveColor` to configure other widgets.  We'll need to replace references to `ThemeData.of(context).toggleableActiveColor` with an expression that looks up the expected ColorScheme color and deals with the difference between `ThemeData.useMaterial3=true` and `ThemeData.useMaterial3=true`.
- Apps that override ThemeData.toggleableActiveColor will need to override ThemeData.radioThemeData instead, and we'll need to create a Dart Fix that makes such a transformation.  It's not clear that this is currently possible - @Piinks what do you think? Something like: 

```dart
ThemeData.light().copyWith(
  toggleableActiveColor: Colors.indigo,
)

// Becomes

ThemeData.light().copyWith(
  radioTheme: RadioThemeData(
    overlayColor: MaterialStateProperty.resolveWith<Color?>((Set<MaterialState> states) {
      return states.contains(MaterialState.selected) ? Colors.indigo : null;
    }),
  ),
)

// Or, if the app has specified other radioTheme properties

ThemeData.light().copyWith(
  toggleableActiveColor: Colors.indigo,
  radioTheme: RadioThemeData(
    splashRadius: 10,
  ),
)

// Becomes

ThemeData.light().copyWith(
  radioTheme: RadioThemeData(
    splashRadius: 10,
    overlayColor: MaterialStateProperty.resolveWith<Color?>((Set<MaterialState> states) {
      return states.contains(MaterialState.selected) ? Colors.indigo : null;
    }),
  ),
)

// Or, worse still

ThemeData.light().copyWith(
  toggleableActiveColor: Colors.indigo,
  radioTheme: MyRadioThemeData(),
)

// Becomes

ThemeData.light().copyWith(
  radioTheme: RadioThemeData(
    splashRadius: 10,
    radioTheme: MyRadioThemeData().copyWith(
      overlayColor: MaterialStateProperty.resolveWith<Color?>((Set<MaterialState> states) {
        return states.contains(MaterialState.selected) ? Colors.indigo : null;
      }),
    ),
  ),
)
```

Reply:
@HansMuller 
Thanks for the feedback. I have looked into existing dart fixes and documentation. 

> Apps that override ThemeData.toggleableActiveColor will need to override ThemeData.radioThemeData instead, and we'll need to create a Dart Fix that makes such a transformation. It's not clear that this is currently possible - @Piinks what do you think? Something like:

I was able to replace `toggleableActiveColor` with `RadioListTheme` however, I am not sure if it is possible to check if `RadioListTile` is present in the code or not, since `toggleableActiveColor` is used multiple widgets, the correct theme should be added. 

My original PR https://github.com/flutter/flutter/pull/95870 which led to this PR. It removes `toggleableActiveColor` without adding a theme.

Could we maybe add more details about list tile widgets changes in migration [guide](https://github.com/flutter/website/blob/main/src/release/breaking-changes/toggleable-active-color.md) for the original PR itself instead of adding theme for each widget using dart fix?

Reply:
>  if only RadioListTile is present as well as toggleableActiveColor in the same code.

Is there to do this currently in dartfix?

Reply:
No, dart fix cannot do this. That code could be in a separate file or directory, there isn't a way for dart fix to know.

Reply:
Hey @HansMuller 

Since `toggleableActiveColor` is also used for widgets other than `RadioListTile`, do you think dart fix to add `RadioListTileData` is still an option here?

Thanks

Reply:
It sounds like the answer is "no" however in this case and others I think the final default should be a ColorScheme color (presumably ColorScheme.secondary)?  Maybe you're planning to remove this reference in a second step?